scout/adapter/mongo/case.py

```diff
@@ -565,9 +565,9 @@ def update_case(self, case_obj):
             {'_id': case_obj['_id']}
         )
 
-        # if case is updated and was archived or active - make it inactive
+        # if case is updated and was archived make it inactive instead
         updated_status = old_case.get('status')
-        if old_case.get('status') in ['archived', 'active']:
+        if updated_status ==  'archived':
```

Comment:
..but then the inactivate-upon-rerun logic fails: rerun active cases will no longer be made inactive. Also, all existing comments would now flag as "old" since the case has indeed been updated. 

What do you think about silent (like minor) update mode for the update function that does not alter case status or modification date? We could then use that also for the report reuploads?

Reply:
Right! A minor update mode or passing a flag to the function! Good point!